dist/index.html

```diff
@@ -3,9 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
```

Comment:
Please change the title to match the app name

Reply:
Done :heavy_check_mark: